examples/hosts/app-integration/external-controller/src/view.ts

```diff
@@ -74,3 +75,22 @@ export function renderAudience(audience: ITinyliciousAudience | IFrsAudience, di
 
     wrapperDiv.appendChild(audienceDiv);
 }
+
+export function renderFocusPresence(focusTracker: FocusTracker, div: HTMLDivElement) {
+    const wrapperDiv = document.createElement("div");
+    wrapperDiv.style.textAlign = "left";
+    wrapperDiv.style.margin = "70px";
+    div.appendChild(wrapperDiv);
+
+    const focusDiv = document.createElement("div");
+    focusDiv.style.fontSize = "14px";
+
+    const onFocusChanged = () => {
+        focusDiv.innerHTML = focusTracker.getPresences().join("<br/>");
```

Comment:
Continuing from other comment, the view here could then decide based off of the presence enum what string content it wants to display for each status